src/validation.cpp

```diff
@@ -1693,13 +1693,14 @@ static bool ConnectBlock(const CBlock& block, CValidationState& state, CBlockInd
 {
     AssertLockHeld(cs_main);
     assert(pindex);
+    const uint256& blockHash = block.GetHash();
```

Comment:
If you're going to stick to variable names that look like Hungarian style (which is certainly not a requirement, IMO), I do prefer it to be consistent. `hashBlock` would be correct here.